astroid/nodes/scoped_nodes.py

```diff
@@ -524,6 +523,8 @@ def __init__(
         """
         self.future_imports = set()
 
+        super().__init__(parent=parent)
```

Comment:
```suggestion
        super().__init__(lineno=1, parent=parent)
```
Mentioned it here: https://github.com/PyCQA/astroid/issues/1273#issuecomment-975510511. Let's try if that works.